config/network/errors.go

```diff
@@ -0,0 +1,30 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/network/p2p"
+)
+
+// ErrInvalidLimitConfig indicates the validation limit is < 0.
+type ErrInvalidLimitConfig struct {
+	// controlMsg the control message type.
+	controlMsg p2p.ControlMessageType
+	err        error
```

Comment:
if we want to be able to interpret this as part of the error chain, need to implement `Unwrap()` method

Reply:
https://github.com/onflow/flow-go/pull/4340/commits/5f52386b4b348f5f6a88e12c9b902cd80ffcef4f